From 7f1808a6e23d6b18ea02f88b09e85ad6515a08ba Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Wed, 5 Apr 2017 17:59:41 +0800
Subject: [PATCH] arm: dts: rk3288-android: use secure efuse

Use a new compatible to match secure interface
when kernel is in no-secure mode.

Change-Id: I3994b2c86bb9e221f102766c2d1a341930628b5d
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm/boot/dts/rk3288-android.dtsi | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/arch/arm/boot/dts/rk3288-android.dtsi b/arch/arm/boot/dts/rk3288-android.dtsi
index cb4a44185e58..0ce5096383dd 100644
--- a/arch/arm/boot/dts/rk3288-android.dtsi
+++ b/arch/arm/boot/dts/rk3288-android.dtsi
@@ -216,6 +216,10 @@
 	reg = <0xff600000 0x4000>;
 };
 
+&efuse {
+	compatible = "rockchip,rk3288-secure-efuse";
+};
+
 &rga {
 	compatible = "rockchip,rga2";
 	clocks = <&cru ACLK_RGA>, <&cru HCLK_RGA>, <&cru SCLK_RGA>;
-- 
2.35.3

